core/c11_atomic.c

```diff
@@ -36,7 +36,7 @@
 #include "irq.h"
 
 /* GCC documentation refers to the types as I1, I2, I4, I8, I16 */
-typedef uint8_t  I1;
+typedef uint8_t I1;
```

Comment:
ups, maybe I should look at the complete file instead of only this single line - never mind :-)